spec/system/file_imports/edit_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe "file_imports/edit" do
+  it "displays edit link", :admin do
+    create :file_import
+    admin = create :admin
+
+    login_as admin
+    visit file_imports_path
+
+    expect(page).to have_link("Edit", exact: true)
+  end
```

Comment:
Since this spec isn't testing any form or clicking link functionality, let's move it to `spec/views`, since those tests run faster. I have an example in that directory you can take a look at.